jobs/config.json

```diff
@@ -147,6 +229,20 @@
   ]
 },
 
+"pull-kubernetes-node-e2e": {
+  "scenario": "kubernetes_kubelet",
+  "args": [
+    "--scenario=test/e2e_node/jenkins/jenkins-pull.properties"
```

Comment:
this looks related to a fatal error I'm seeing in CI here: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/39871/pull-kubernetes-node-e2e/12862/